changelog.txt

```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
+- Functionality on Unix with Xamarin.Mac Full profile and Mono (#451)
+- Handle new RSACng default on netfx4.7 (#448)

```

Comment:
Rephrase this to, "Added support to use RSACng in .net 4.7"

Reply:
ok thanks